.github/workflows/test.yml

```diff
@@ -260,6 +260,13 @@ jobs:
           os: macos-13
           arch: x86_64
           action: test
+        - name: 'WAMR on Windows/x86_64'
```

Comment:
Nit: Tests are sorted alphabetically by engine and then OS, so this should be further up.